web-registry/src/components/organisms/BasketOverview.tsx

```diff
@@ -0,0 +1,274 @@
+
+interface BasketOverviewProps {
+  basketDenom: string;
+}
+
+const BasketOverview: React.FC<BasketOverviewProps> = ({ basketDenom }) => {
+  const styles = useStyles();
+  const [basket, setBasket] = useState<Basket>();
+
+  useEffect(() => {
+    if (!basketDenom || !ledgerRestUri) return;
+
+    const fetchData = async (): Promise<void> => {
+      const _basket = await getBasket(basketDenom);
+      setBasket(_basket);
+    };
+
+    fetchData();
+  }, [basketDenom]);
+
+  return (
+    <Box className={styles.top}>
+      <Section className={styles.content}>
+        <Grid container>
+          <Grid item xs={12} sm={5} className={styles.imageContainer}>
+            <OptimizedImage
+              className={styles.image}
+              src={forestImg}
+              alt={basket?.name}
+            />
+          </Grid>
+          <Grid item xs={12} sm={7} className={styles.textContainer}>
+            <Title variant="h1" className={styles.title}>
+              {basket?.name}
+            </Title>
+            <Description className={styles.basketDenom}>
+              {basket?.basket_denom}
+            </Description>
+            <Description className={styles.basketDescription}>
+              {/* TODO: Hardcoded description */}
+              {basketDescription}
+            </Description>
+            <OnBoardingCard className={styles.card}>
+              <Grid
+                container
+                rowSpacing={1}
+                columnSpacing={{ xs: 1, sm: 2, md: 3 }}
+              >
+                {basketSummary.map(item => (
+                  <Grid item key={item.id} xs={12} sm={6}>
+                    <Item
+                      label={item.displayName}
+                      data={item.value}
+                      link={item.link}
+                    />
+                  </Grid>
+                ))}
+              </Grid>
+            </OnBoardingCard>
+          </Grid>
+        </Grid>
+      </Section>
+    </Box>
+  );
+};
+
+export { BasketOverview };
+
+/**
+ * Basket summary item (subcomponent)
+ */
+
+const useStylesItem = makeStyles(theme => ({
+  gridItem: {
+    marginTop: theme.spacing(4),
+  },
+  label: {
+    fontSize: theme.typography.pxToRem(12),
+    color: theme.palette.primary.contrastText,
+    letterSpacing: '1px',
+    lineHeight: '15px',
+    marginBottom: theme.spacing(2),
+  },
+  data: {
+    fontSize: theme.typography.pxToRem(16),
+    fontWeight: 400,
+    '& a': {
+      color: theme.palette.text.secondary,
+      fontSize: theme.typography.pxToRem(16),
+      fontWeight: 400,
+    },
+  },
+  arrowIcon: {
+    marginLeft: theme.spacing(1),
+    marginBottom: theme.spacing(0.3),
+    height: 9,
+    width: 13,
+  },
+}));
+
+interface ItemProps {
+  label: string;
+  data: string | JSX.Element;
+  link?: string;
+}
+
+const Item = ({ label, data, link }: ItemProps): JSX.Element => {
+  const styles = useStylesItem();
+
+  return (
+    <div className={styles.gridItem}>
```

Comment:
To put my longer comment in context with an example, this element could be rewritten as
```suggestion
    <Box cx={{ mt: 4 }}>
```

and you could remove the `gridItem` styles. 

alternatively, you could make a 
```tsx
const GridItem = styled('div')(({ theme }) => ({
  marginTop: theme.spacing(4),
}))
```

and change this line to 
```suggestion
    <GridItem>
```

but that's the same amount of boilerplate, so I prefer `sx` for styles that aren't reused (it's a good bit less code)

To be clear with these comments: I'm not saying you should remove all of the `makeStyles`s in this story (you would probably still need some to pass `classes` to components like the `<Section>` element in `BasketOverview`), and am fine with folks using `makeStyles` if that's the preference, but do think it could create some dependency friction in the future when we go to upgrade

Reply:
Wow, I've just been testing a bit and it's really tricky. Now I am going to make the minimum changes, since the adaptation of existing components is not so strict, and we will make this topic a priority for the coming weeks. Let's analyze the different cases and agree among all the best ways to apply the styles and the changes now to prepare for the following refactorings to come. Thanks, David!!